prime-router/src/main/kotlin/Report.kt

```diff
@@ -76,6 +76,7 @@ data class ReportStreamFilterResult(
     val filterArgs: List<String>,
     val filteredTrackingElement: String,
     val filteredIndex: Int,
+    val filterType: ReportStreamFilterType?,
```

Comment:
@whytheplatypus 
this seems to need to remain `null`able for backwards compatibility...
if i attempt to get a submission history without the `filterType` populated in the serialized JSON, i get this error without `filterType` being `null`able
```
Caused by: com.fasterxml.jackson.module.kotlin.MissingKotlinParameterException: Instantiation of [simple type, class gov.cdc.prime.router.ReportStreamFilterResult] value failed for JSON property filterType due to missing (therefore NULL) value for creator parameter filterType which is a non-nullable type
```

open to suggestions on how to do this without making it non-nullable :smile: